package.json

```diff
@@ -96,9 +96,9 @@
     "devtools:redux": "remotedev --hostname=localhost --port=8000",
     "start:dev": "concurrently -k -n build,react,redux yarn:start yarn:devtools:react yarn:devtools:redux",
     "announce": "node development/announcer.js",
-    "storybook": "storybook dev -p 6006 -c .storybook",
-    "storybook:build": "storybook build -c .storybook -o storybook-build",
-    "storybook:deploy": "storybook-to-ghpages --existing-output-dir storybook-build --remote storybook --branch master",
+    "storybook": "STORYBOOK_ENV=true storybook dev -p 6006 -c .storybook",
```

Comment:
In the case of running one of these Storybook-related commands, the `STORYBOOK_ENV` is set to true